src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -15,6 +15,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <Import Project="$(MSBuildThisFileDriectory)Microsoft.NET.DefaultAssemblyInfo.targets" />
```

Comment:
Good catch. I guess imports are already relative to current dir, this is unnecessary and evaluating to empty, which still works. I'll correct the typo and keep it this way for consistency but we should probably remove the unnecessary uses of MSBuildThisFileDirectory for readability at some point.